app/lib/version_change.rb

```diff
@@ -0,0 +1,28 @@
+class VersionChange
+  def resources
+    {}
+  end
+
+  def self.description(text)
+    define_method(:description) { text }
+  end
+
+  def self.action(controller, controller_action)
+    define_method(:actions) do
```

Comment:
discussed in person: not a blocker but would be great if we were to extend this to support multiple actions/controllers. But can leave till we need it